BetterMenus/Blueprints/UpdateVisibility.cs

```diff
@@ -9,6 +9,12 @@
 namespace Satchel.BetterMenus
 {
     public static partial class Blueprints{
+        /// <summary>
+        /// a blueprint to help update visibility of many elements
+        /// </summary>
+        /// <param name="menuRef">the Menu instance that created that holds the elements in the menu</param>
+        /// <param name="isVisible">the new visibility of the elements</param>
+        /// <param name="elements">an array of the ids of the elements that want to chance its visibility</param>
```

Comment:
an array of the ids of the elements that want to chance its visibility
=>
array of element ids to update